tests/test_dbt_parser.py

```diff
@@ -56,17 +55,18 @@ def test_get_models(self):
         mock_self.get_dbt_selection_models.assert_called_once_with(selection)
         self.assertEqual(models, mock_return_value)
 
-    def test_get_models_unsupported_manifest_version(self):
+    def test_get_models_simple_select(self):
         mock_self = Mock()
         mock_self.project_dir = Path()
         mock_self.dbt_version = "1.4.0"
```

Comment:
Can you mock this with these too to capture all the major manifest versions? 
https://docs.getdbt.com/reference/artifacts/manifest-json

- 1.0
- 1.1
- 1.2
- 1.3